src/renderers/WebGLRenderer.js

```diff
@@ -645,6 +645,12 @@ class WebGLRenderer {
 
 			animation.stop();
 
+			if ( typeof __THREE_DEVTOOLS__ !== 'undefined' ) {
+
+				__THREE_DEVTOOLS__.dispatchEvent( new CustomEvent( 'dispose', { detail: this } ) );
```

Comment:
AFAIK, are not further developed anymore so they won't use the `dispose()` event.

The renderer should only emit this event if it is actually consumed by the three dev tools.

Reply:
Yea I figured, however I know [Three Tools](https://three.tools/) uses this and it was mostly for consistency, similar to `Texture`. I mostly wanted the `Scene` to dispatch when it gets disposed  https://github.com/mrdoob/three.js/blob/ac95b71239d45ae9fd489d6e13b2729803d97143/src/textures/Texture.js#L209